CHANGELOG.md

```diff
@@ -6,7 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-Nothing yet...
+### Added
+ - New linters
+  - ScenarioWithBadNameLinter
```

Comment:
For the naming convention of linters, I have been using 'test' for linters that are applicable to both scenarios and outlines (e.g. TestWithNoNameLinter) and using 'element' for linters that are applicable to many different model types (e.g. ElementWithTooManyTagsLinter). We don't have to use the original name. Especially if a linter's behavior or scope changes when it gets updated. So, in this case, please use `TestWithBadNameLinter`.